test/Tranche.t.sol

```diff
@@ -306,9 +305,9 @@ contract DepositAndWithdrawalTest is TrancheTest {
         address owner,
         address receiver
     ) public {
-        // Given: assetsDeposited bigger than 0, bigger than equal to assetsWithdrawn, receiver is not pool or liquidityProvider
+        // Given: assetsDeposited bigger than 0 and assetsWithdrawn, receiver is not pool or liquidityProvider
         vm.assume(assetsDeposited > 0);
-        vm.assume(assetsDeposited >= assetsWithdrawn);
+        vm.assume(assetsDeposited > assetsWithdrawn);
```

Comment:
Don't change test because there is an error (in this case I would expect a divide by 0)! this is something we should solve in the contract itself.

Example check if it is the first liquidity deposited in a pool, if yes, burn a small amount, let's discuss tomorrow in stand-up